client/landing/stepper/declarative-flow/internals/steps-repository/goals/index.tsx

```diff
@@ -81,35 +84,41 @@ const GoalsStep: Step = ( { navigation } ) => {
 		recordTracksEvent( 'calypso_signup_goals_select', eventProperties );
 	};
 
-	const recordIntentSelectTracksEvent = (
-		submittedGoals: Onboard.SiteGoal[],
-		intent: Onboard.SiteIntent
-	) => {
-		const hasImportGoal = submittedGoals.includes( SiteGoal.Import );
-
+	const recordIntentSelectTracksEvent = ( intent: Onboard.SiteIntent ) => {
 		const eventProperties = {
 			intent,
-			import: hasImportGoal,
```

Comment:
Removing properties can break existing funnels. I did a non-exhaustive search of the existing funnels which use `calypso_signup_intent_select` and didn't find anything using the `import` prop (specifically filtering for funnels created by those involved in [the original PR](https://github.com/Automattic/wp-calypso/pull/64725)), so maybe this is safe to do? But if there's no pressing reason to drop the prop, maybe we should keep it?

Reply:
I'm not sure keeping it helps much 🤷🏻  It will always be `false` since we were considering passing an empty array (and completely removed the parameter as of my last commit).  

Anyway, I went through all funnels containing the event, and the `import` property specifically filters none of them, so it's safe to ignore it. We have created only one funnel this year containing the event, and the last-but-one was calculated 12 months ago. It's a pretty safe ignore, IMHO.

Reply:
Oh yeah, always false, you're right 😅